xmille/curses_ui.c

```diff
@@ -13,6 +13,8 @@
 #endif
```

Comment:
I think you should probably leave the curses bits alone; they aren't even compiled for xmille, and so changes may well break things that will not be detected. I don't think we want to try and make them work at this point.